Extensions/xkit_patches.js

```diff
@@ -160,6 +160,22 @@ XKit.extensions.xkit_patches = new Object({
 			XKit.tools.Nx_XHR = function(details) {
 				details.timestamp = new Date().getTime() + Math.random();
 
+				const standard_headers = {
+					"X-Requested-With": "XMLHttpRequest",
+					"X-Tumblr-Form-Key": XKit.interface.form_key()
+				};
+
+				if (details.headers == undefined) {
```

Comment:
i don't see how it could be a linter error? there's no technical reason this key couldn't be set to `null`, and loosely comparing to `undefined` seems preferable to testing for ANY falsy value just for readabilty.

i'm not at all opposed to changing to `===` here but it'd be hollow if i did that without understanding why testing for "is this key undefined" is that much worse than "is this key *exactly* undefined"

Reply:
the answer is that no-one can remember how `==` coercion works in JS off the top of their head, so we should just stop using it where we can. https://eslint.org/docs/rules/eqeqeq

(for example, when I first made this comment, I had thought that comparing to undefined did coercion to falsey. I was mistaken, it's only `== false` that does that (unless the value is undefined, in which case `undefined == false` will return false))